common/src/main/java/org/dash/wallet/common/services/analytics/AnalyticsService.kt

```diff
@@ -17,16 +17,16 @@
 
 package org.dash.wallet.common.services.analytics
 
-import android.os.Bundle
 import android.util.Log
+import androidx.core.os.bundleOf
 import com.google.firebase.analytics.ktx.analytics
 import com.google.firebase.crashlytics.ktx.crashlytics
 import com.google.firebase.ktx.Firebase
 import org.dash.wallet.common.BuildConfig
 import javax.inject.Inject
 
 interface AnalyticsService {
-    fun logEvent(event: String, params: Bundle)
+    fun logEvent(event: String, params: Map<AnalyticsConstants.Parameter, Any>)
```

Comment:
The point of this change it to keep parameters limited to predefined values that we can create custom definitions for. That way, they'll show up on the event page like here:
<img width="507" alt="Screenshot 2023-03-16 at 14 45 21" src="https://user-images.githubusercontent.com/7733989/225824848-d38af7a0-b130-4cec-a851-e411cbe64e03.png">

Reply:
very cool.